### DISCUSSION TITLE
Checking for new posts

### DISCUSSION THREAD

**Discussion by mcl-playz** [discord-jda/JDA#2511] [Questions and Help] [answered]
How would i check when a post is created in a specific forum. I just need it to send a message in that post.

**Comment by MinnDevelopment** [comment, marked as answer]
In the [MessageReceivedEvent](https://docs.jda.wiki/net/dv8tion/jda/api/events/message/MessageReceivedEvent.html), you need to check that the message came from a thread channel, and then check the parent channel type:

```java
event.isFromThread() && event.getChannel().asThreadChannel().getParentChannel().getType() == ChannelType.FORUM
```

**Reply by mcl-playz** [reply]
Thanks
